src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugBridge.cs

```diff
@@ -157,20 +184,26 @@ public GethLikeTxTrace[] GetBlockTrace(Rlp blockRlp, CancellationToken cancellat
             return _tracer.TraceBlock(blockRlp, gethTraceOptions ?? GethTraceOptions.Default, cancellationToken);
         }
 
-        public byte[] GetBlockRlp(Keccak blockHash)
-        {
-            return _dbMappings[DbNames.Blocks].Get(blockHash);
-        }
 
         public byte[] GetBlockRlp(long number)
         {
             Keccak hash = _blockTree.FindHash(number);
             return hash == null ? null : _dbMappings[DbNames.Blocks].Get(hash);
         }
+        public byte[] GetBlockRlp(Keccak blockHash)
+            => _dbMappings[DbNames.Blocks].Get(blockHash);
```

Comment:
This two share code

Reply:
should I change the first function to depend on the second ?